smartmin/csv_imports/models.py

```diff
@@ -19,6 +19,14 @@ def generate_file_path(instance, filename):
 
 
 class ImportTask(SmartModel):
+    PENDING = 'PENDING'
+    STARTED = 'STARTED'
+    RUNNING = 'RUNNING'
+    SUCCESS = 'SUCCESS'
+    FAILURE = 'FAILURE'
+
+    READY_STATES = [SUCCESS, FAILURE]
```

Comment:
Just realized you might be using these to keep compatibility with what the Celery states are. If so then that's fine.